### DISCUSSION TITLE
Bug For Repo Commits Feed

### DISCUSSION THREAD

**Discussion by Colin-XKL** [community/community#86096] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

When I try to subscribe a repo's commits via atom feed, my feed reader got error when fetching the atom file. I checked the link and found that it didn't returned an atom xml file as expected, it returned a json file. 

When accessing the url in browser, the result is a xml file. But when using curl or other cli tool, the result is a json file. This seems to be a bug, and only affects repo's commit atom link. 

sample URL: https://github.com/textpattern/textpattern/commits.atom

```shell
# this returns a xml file
curl https://github.com/textpattern/textpattern/commits.atom -H 'Accept: application/atom+xml'

# but this one, without specify the 'Accept' header, we got a json file
curl https://github.com/textpattern/textpattern/commits.atom  


# for other atom link, it works as expected.
curl https://github.com/textpattern/textpattern/releases.atom # returns xml
curl https://github.com/textpattern/textpattern/tags.atom # returns xml

```

**Comment by apraile** [comment]
Related:

- newsboat issue: https://github.com/newsboat/newsboat/issues/2659
- Miniflux issue: https://github.com/miniflux/v2/issues/2293

**Comment by masnagam** [comment]
Now `curl https://github.com/textpattern/textpattern/commits.atom` seems to return atom xml.  In fact, I can see contents of a github commit feed in Miniflux.

**Comment by hossinhabeeb119** [comment]
Rt

**Comment by Cbodfield** [comment, marked as answer]
Appreciate the feedback everyone! We shipped a fix for this yesterday. ❤️
